Sources/Charcoal/CharcoalViewController.swift

```diff
@@ -250,6 +262,17 @@ extension CharcoalViewController: FilterViewControllerDelegate {
         viewController.delegate = self
         pushViewController(viewController, animated: true)
     }
+
+    private func pushTabBarController(_ controller: UITabBarController) {
+        controller.hidesBottomBarWhenPushed = true
+        pushViewController(controller, animated: true)
+    }
+
+    private func showPolygonSearchCalloutIfNeeded() {
+        guard !UserDefaults.standard.polygonSearchCalloutShown else { return }
+        UserDefaults.standard.polygonSearchCalloutShown = true
+        showCalloutOverlay(withText: "map.polygonSearch.callout.title".localized(), andDirection: .up, andArrowAlignment: .right(15), constrainedToDirectionalAnchor: navigationBar.bottomAnchor)
```

Comment:
in cases where methods get too long horizontally, it's always nice to split them in several lines, it's much more readable. This applies in several parts of this PR, which is not that critical but more a style preference (at least for me)